domains/_discord.example.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "github-username",
+        "email": "email@address"
```

Comment:
You should replace `email@address` with your email address (or keep as empty string, `""`)